hybrid/multi-region-clusters/internal-listeners/networking/gke/networking-GKE-README.md

```diff
@@ -82,7 +82,7 @@ Namespace:                kube-system
 ...
 Selector:                 k8s-app=kube-dns
 Type:                     LoadBalancer
-IP:                       10.0.37.50
```

Comment:
Is this an intentional change?

Reply:
A great catch, @shreedakv! I restored this file.